src/omni-slider.js

```diff
@@ -1,532 +1,540 @@
-/* Constructor function
- * elementContainer - this acts as a wrapper for the slider, all of the sliders DOM elements will be transcluded inside the <element> provided
- * options - contains the options for the slider
- */
-function Slider(elementContainer, options) {
+(function() {
```

Comment:
Is it possible to keep the gulp took support IIFE, then the code base would get the benefit of following es2015 pattern natural without being wrapped in IIFE?